CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Given a version number MAJOR.MINOR.PATCH, increment:
 ## [Unreleased]
 ### Added
 - pictureUrl attribute to DynamicBrcode resource
+- picture to update method in Workspace resource
+- status, organizationId, pictureUrl and created attributes to Workspace resource
```

Comment:
Faltaram alguns pontos em Added

- updated, type and transactionIds attribute to UtilityPayment resource
- workspaceId attribute to Boleto resource
- updated attribute to BoletoHolmes.Log resource
- description attribute to PaymentRequest resource
- amount and transactionIds attribute to BoletoPayment resource
- workspaceId attribute to Event resource
- transactionIds attribute to DarfPayment and TaxPayment resources

Reply:
Falta colocar que removeu o recurso BrcodePreview deprecado

Reply:
Esses que eu comentei que precisa adicionar, alguns não foram adicionados no sdk ainda.